go.mod

```diff
@@ -1,26 +1,27 @@
 module github.com/ipfs-shipyard/vole
 
-go 1.23
+go 1.24
+
+toolchain go1.24.2
```

Comment:
IIUC we should remove this to prevent conflicts, right? I don't see the toolchain in some of the other binaries we maintain (e.g. kubo and rainbow)